src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -408,15 +396,21 @@ private void esbRestApiCall(HttpServletRequest request,
 	 * using Apache Http client library
 	 */
 
-
 	private void apiConnectionLogic(HttpServletRequest request, HttpServletResponse response)
 			throws IOException {
 		PrintWriter out = response.getWriter();
-		if(request.getPathInfo().equalsIgnoreCase(MPATHWAYS_PATH_INFO)){
-			mpathwaysCall(request, response, out);
+		String isStubTesting = appExtPropertiesFile.getProperty(SectionUtilityToolFilter.PROPERTY_TEST_STUB);
+		M_log.debug("isStubTesting: " + isStubTesting);
+		if(Boolean.valueOf(isStubTesting)){
```

Comment:
This is better done by using an interface with two implementations, but that can be left for another time.